pages/api/locker/[...lockerId].ts

```diff
@@ -0,0 +1,31 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+import prisma from '../../../lib/prisma'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'GET') {
+    try {
+      let { lockerId } = req.query
```

Comment:
this should be immutable
```suggestion
      const { lockerId } = req.query
```